client/src/components/EntryButtonGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, WhiteSpace } from 'antd-mobile';
+
+const EntryButtonGroup: React.FC = () => {
+  return (
+    <Div>
+      <Button
+        type="primary"
+        onClick={() => {
+          location.href = '/user';
```

Comment:
location.href를 이용하면 reload되면서 페이지가 이동하지 않나요? react-router-dom의 history를 이용하면 어떨까요?
```suggestion
import React from 'react';
import { useHistory } from 'react-router-dom';
import styled from 'styled-components';
import { Button, WhiteSpace } from 'antd-mobile';

const EntryButtonGroup: React.FC = () => {
  const history = useHistory();

  return (
    <Div>
      <Button
        type="primary"
        onClick={() => {
          history.push('/user');
```

Reply:
a7138da : 리뷰 참고하여 고쳤습니다  !  👍